lava_test_plans/projects/lkft-android/include/metadata.jinja2

```diff
@@ -0,0 +1,22 @@
+  android.build: "{{ BUILD_NUMBER }}"
+  android.name: "{{ JOB_NAME }}"
+  android.url: "{{ REFERENCE_BUILD_URL }}"
+  android.version: "{{ ANDROID_VERSION }}"
```

Comment:
isn't "android." unnecessary ?

Reply:
it's been used with the android prefix since the beginning, and I really think this makes it more clear to distinguish with others.

Reply:
ok